content/agent/kubernetes/helm.md

```diff
@@ -111,39 +111,52 @@ To install the chart with the release name `<RELEASE_NAME>`, retrieve your Datad
 helm install --name <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY> stable/datadog
 ```
 
-Refer to the section below for a list of configurable parameters.
-
 This chart adds the Datadog Agent to all nodes in your cluster via a DaemonSet. It also optionally deploys the [kube-state-metrics chart][5] and uses it as an additional source of metrics about the cluster. A few minutes after installation, Datadog begins to report hosts and metrics.
 
-### Configuration
+**Note**: For a full list of the Datadog chart's configurable parameters and their default values, refer to the [Datadog Helm repository README][6]. 
+
+## Configuring the Datadog Helm chart
 
-For a full list of the Datadog chart's configurable parameters and their default values, refer to the [Datadog Helm repository readme][6].
+As a best practice, a YAML file that specifies the values for the chart parameters should be provided to configure the chart: 
```

Comment:
not sure it's a best practice, I think this is a required one ?

Reply:
It's basically required if you want to know what has been running (so you re-apply the same settings when upgrading) but in theory one could have a bash script running `helm` with three lines of `--set`.

Reply:
I hesitated to document that.. but moving forward with all the config options I think we shouldn't advocate for it. 

Having a one simple line bash command to first launch the Agent is fine, it makes onboarding easier, testing easier...  But then using the `value.yaml` logic is more heathy and allows for more consistency over the configuration options documentation. what do you think?

Reply:
Perfect, we keep the one-liner for easy testing, but move people towards a `values.yaml` file when they want to customize

Reply:
sounds good to me